cobalt/loader/image/animated_image_tracker.cc

```diff
@@ -140,13 +141,27 @@ void AnimatedImageTracker::Reset() {
 }
 
 void AnimatedImageTracker::OnRenderTreeRasterized() {
+  bool is_main_webmodule = name_ == "MainWebModule";
   count_animated_images_active = playing_urls_.size();
+  if (is_main_webmodule) {
+    UmaHistogramExactLinear("AnimatedImage.MainWebModule.Active",
```

Comment:
Please follow the naming convention: `Cobalt.SubCategory...Name` per go/cobalt-telemetry#histograms. Cobalt as the root namespace will help me disentangle Cobalt metrics from everything else. IDC how deeply nested your sub categories are, but Cobalt should be the root.

Reply:
Yeah, i was staring at this and didn't know what to put there. Do you have a quick draft doc that outlines the convention ? We might even do a pre-submit check for following the convention.

Reply:
go/cobalt-telemetry#histograms would be the doc for defining this. I did make an attempt at it there.

Generally, just think of it as namespacing where each subsequent namespace adds another layer of specificity. You'll only add new subcategories when it's necessary to disambiguate from different but similar measurements.